package.json

```diff
@@ -7,40 +7,40 @@
     "url": "https://github.com/mozilla/chronicle/issues"
   },
   "dependencies": {
-    "bell": "2.0.0",
+    "bell": "2.1.0",
     "boom": "2.6.1",
     "commander": "2.6.0",
-    "convict": "0.6.0",
-    "elasticsearch": "3.1.1",
-    "embedly": "1.0.2",
-    "hapi": "8.1.0",
+    "convict": "0.6.1",
+    "elasticsearch": "3.1.3",
+    "embedly": "1.0.4",
+    "hapi": "8.2.0",
     "hapi-auth-cookie": "2.0.0",
     "joi": "5.1.0",
-    "mozlog": "1.0.1",
-    "node-resque": "0.11.5",
+    "mozlog": "1.0.2",
+    "node-resque": "0.11.6",
     "nodemailer": "1.3.0",
     "nodemailer-smtp-transport": "0.1.13",
-    "pg": "4.1.1",
+    "pg": "4.2.0",
```

Comment:
yay! pg has a changelog for minor releases: https://github.com/brianc/node-postgres/blob/master/NEWS.md

- Support for additional error fields in postgres >= 9.3 if available